File: epargne/ExternalApi/DTO/ClientCourantDto.cs
using System;

namespace Epargne.ExternalApi.DTO
{
    public class ClientCourantDto
    {
        public int IdClient { get; set; }          // équivalent de Integer idClient
        public string Nom { get; set; }            // équivalent de String nom
        public string Prenoms { get; set; }        // équivalent de String prenoms
        public string DateNaissance { get; set; }  // String ou format ISO (yyyy-MM-dd)
    }
}

File: server-api/src/main/java/com/example/server_dtos/ClientCourantDto.java
package com.example.server_dtos;

import java.io.Serializable;
import java.time.LocalDate;
import java.util.List;

import lombok.Data;

@Data
public class ClientCourantDto implements Serializable {
    private Integer idClient;
    private String nom;
    private String prenoms;
    private String email;
    private String mdp;
    private LocalDate dateNaissance;
    private Integer idDirection;
    private List<ClientRoleDto> clientRoles;
}
